internal/bminventory/inventory.go

```diff
@@ -3955,7 +3957,9 @@ func validateArchitectureAndVersion(v versions.Handler, c *common.Cluster, cpuAr
 		ocpVersion = c.OpenshiftVersion
 	}
 	if c.CPUArchitecture != common.MultiCPUArchitecture {
-		if c.CPUArchitecture != "" && c.CPUArchitecture != cpuArch {
+		if c.CPUArchitecture == "aarch64" && cpuArch == "arm64" {
```

Comment:
Should not fix the arch to x86 and arm, the original check should work for multi-arch support.

Reply:
@cs-zhang I moved this logic out of the MultiCPUArchitecture block. It needs to be there otherwise I get a `release = aarch64 and cpu arch = arm64 does` not match error